### PR TITLE
chore(ci): do not parse report dir when walking subdirectories

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: ci/benchmark_parser.py [hunk 1]
```diff
@@ -52,6 +52,8 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix=""):
             continue
         for subdir in dire.iterdir():
             if walk_subdirs:
+                if subdir.name == "report":
+                    continue
                 subdir = subdir.joinpath("new")
             elif subdir.name != "new":
```
Comment (IceTDrinker) [inline ci/benchmark_parser.py:58]: what about checking if subdir exists ? and if it does not continue ? Handles all the potential dirs in there

Comment (soonum) [reply ci/benchmark_parser.py:58]: I don't quite understand your point.

Comment (IceTDrinker) [reply ci/benchmark_parser.py:58]: her our issue is that report/new did not exist so my proposal is to do

```python
subdir = subdir.joinpath("new")
if not subdir.exists():
    continue
```

this way we avoid all dirs that may not be benchmark dirs

what do you think ?

Comment (soonum) [reply ci/benchmark_parser.py:58]: Oh you're right.

**Review by IceTDrinker — APPROVED** [review]
thanks !

**Pull request merged**